### ISSUE TITLE
Login button in closed-wind-down is broken

### ISSUE THREAD

**Issue by aspiers** [swapmyvote/swapmyvote#484]
`onclick="$('#js-login-modal').modal(); return false"` results in:

```
(index):68 Uncaught TypeError: $(...).modal is not a function
    at HTMLAnchorElement.onclick ((index):68)
```

This is super weird because it works fine in other modes, and `.modal()` is defined by bootstrap.

**Comment by JFernando122** [comment]
I have a fix for this, just waiting for #485 to be merged to open the pull request with the changes

**Comment by aspiers** [comment]
Wow great, thanks! #485 merged now.
